examples/chain/chain.yml

```diff
@@ -57,12 +57,12 @@ insertmolecules:
 
 energy:
     - bonded: {}
-    - nonbonded_splined:
+    - nonbonded_exact:
```

Comment:
It is not deprecated according to the manual. However I would prefer to shift the meanings of nonbonded energy keywords in a following way: The `nonbonded_exact` keyword would instruct Faunus which particular method to use (here an exact computation) while `nonbonded` would be just an alias for some concreate method; the aliasʼs target could change between releases or it could even depent on nonbodend interaction used (Faunusʼs educated guess of the most suitable implementation).